src/System.CommandLine.Tests/SuggestDirectiveTests.cs

```diff
@@ -22,11 +24,11 @@ public class SuggestDirectiveTests
 
         public SuggestDirectiveTests()
         {
-            _fruitOption = new Option<string>("--fruit")
-                .WithSuggestions("apple", "banana", "cherry");
+            _fruitOption = new Option<string>("--fruit");
+            _fruitOption.Argument.SuggestionSources.Add( new[] { "apple", "banana", "cherry" });
```

Comment:
This is the one place that feels like a take-back in the API. I could add a `AddSuggestions` to `Option` to improve this.

Reply:
I like that idea.